anta/runner.py

```diff
@@ -282,7 +283,7 @@ async def main(  # noqa: PLR0913
                 "Please consult the ANTA FAQ."
             )
 
-        coroutines = get_coroutines(selected_tests, manager)
+        coroutines = get_coroutines(selected_tests, manager if dry_run else None)
```

Comment:
lets check if this was introduced after 1.1- remove manager from get_coroutine and do the same for loop in dry_run

Reply:
ok this was added in 1.1 and was breaking .. so this fixes it technically

I think for v2.0.0 we need to think about take a decision on how the RM behaves (to quote @mtache maybe we don't want to have to give a RM to the runner)

Reply:
Correct. There is https://github.com/aristanetworks/anta/issues/837 tracking this.